docs/learning/howto/use-roi-metrics.md

```diff
@@ -0,0 +1,202 @@
+1. On the Workflow Tab add a single `Command` step with the command `sleep 5`.  (Assumes this will be running on Linux host)
+1. Choose the _Execution Plugins_ tab.
+1. Check the box for **ROI Metrics Data**.<br><br>
+    ![ROI Metrics Plugin](~@assets/img/roi-metrics-data.png)
+1. Click **Add Custom Field**.<br><br>
+    ~[Add Custom Field](~@assets/img/roi-metrics-customfield.png)
+1. Enter a value for:
+    - ***Field Label***: `Hours Saved`
+    - ***Field Key***: `hours`
+    - ***Description***: `FTE hours saved each time this job is run.`
+1. Click **Add**.
+1. Enter a value of `1.25` for the metric in the **Hours Saved** field.
+1. Save the job.
+1. Run the job and look for the ROI Metrics Data tab.  Confirm that the data is populated.
+1. Run the job a few more times so we have multiple executions.
+
+:::
+::: tab Download Job Definition
+
+Copy the Job Definition below and paste into a plain text file.  Then import to your Process Automation server.  (Version 4.7.0 minimum)
+
+```
+- defaultTab: nodes
+  description: This is an example ROI Job as detailed in the Learning Article on our
+    [docs.rundeck.com](https://docs.rundeck.com) site.
+  executionEnabled: true
+  id: 9616fe19-a8a7-4f44-b785-822c81dddeed
+  loglevel: INFO
+  name: ROI Job
+  nodeFilterEditable: false
+  plugins:
+    ExecutionLifecycle:
+      roi-metrics:
+        userRoiData: '[{"key":"hours","label":"Hours Saved","value":"1.25","desc":"FTE
+          hours saved each time this job is run. (Field key: hours)"}]'
+  scheduleEnabled: true
+  schedules: []
+  sequence:
+    commands:
+    - exec: sleep 5
+    - exec: echo "Saving hours every run"
+    keepgoing: false
+    strategy: node-first
+  uuid: 9616fe19-a8a7-4f44-b785-822c81dddeed
+
+```
+
+:::
+::::
+
+## Gathering Data
+
+>This section assumes familiarity with fundamental Rundeck operations like Creating User API Tokens, editing/importing jobs, etc.
+
+:::: tabs
+::: tab Setup Steps
+
+- Configure a [User API Key](/manual/10-user.md#user-api-tokens) with the necessary access to gather metrics from the job/project where your data is located.
+- Save the User API Key in the Key Storage by going to **Project Settings** > **Key Storage** and adding the key there.
+- Copy the job definition (next tab) and import to your project.
+- Edit the Job Definition.  Configure the `api-key` Option Input with the API Key entry you created earlier.
+- Feel free to update any defaults on the other job options for `rundeck-server`, `project-name`, `job-tags` as needed.
+
+Run the job to gather data and see a list of JSON output for any jobs with ROI data.
+
+:::
+::: tab Gather Metrics Job Definition
+
+Copy the Job Definition below and paste into a plain text file.  Then import to your Process Automation server.  (Version 4.7.0 minimum)
+
+>Note: This is only an example job to show the different calls to make to get a list of ROI data.  This script is not officially supported.
+
+```
+- defaultTab: output
```

Comment:
same thing: i'd prefer we save these in a "sample jobs" repo and provide a link that auto-downloads the job definition. something nearly identical was done [here](https://docs.rundeck.com/docs/learning/solutions/automated-diagnostics/getting-started.html#download-the-automated-diagnostics-solution-package)